cassandane/tiny-tests/JMAPContacts/card_set_create_basic

```diff
@@ -9,67 +9,46 @@ sub test_card_set_create_basic
 
     my $service = $self->{instance}->get_service("http");
     $ENV{DEBUGDAV} = 1;
-    my $carddav = Net::CardDAVTalk->new(
-        user => 'cassandane',
-        password => 'pass',
-        host => $service->host(),
-        port => $service->port(),
-        scheme => 'http',
-        url => '/',
-        expandurl => 1,
-    );
+    my $carddav = $self->default_user->carddav;
 
     xlog $self, "create addressbook";
-    my $res = $jmap->CallMethods([
-            ['AddressBook/set', { create => { "1" => {
-                            name => "foo"
-             }}}, "R1"]
-    ]);
-
-    my $abookid = $res->[0][1]{created}{"1"}{id};
+    my $abook = $self->default_user->addressbooks->create({ name => 'Foo' });
 
     my $now = DateTime->now();
     my $created = $now->strftime('%Y-%m-%dT%H:%M:%SZ');
     my $prodid = '-//Example Corp.//CardDAV Client//EN';
     my $name = 'Mr. John Q. Public, Esq.';
     my $uid = 'ae2640cc-234a-4dd9-95cc-3106258445b9';
 
-    $res = $jmap->CallMethods([
-        ['ContactCard/set', {
-            create => {
-                "1" => {
-                    '@type' => 'Card',
-                    version => '1.0',
-                    addressBookIds => { $abookid => JSON::true },
-                    uid => $uid,
-                    prodId => $prodid,
-                    kind => 'individual',
-                    created => $created,
-                    name => { full => $name }
-                }
-            }
-        }, 'R1']
-    ]);
-
-    $self->assert_not_null($res->[0][1]{created}{1});
-    $self->assert_not_null($res->[0][1]{created}{1}{id});
-
-    my $href = $res->[0][1]{created}{1}{'cyrusimap.org:href'};
-    $res = $carddav->Request('GET', $href, '',
+    my $card = $abook->create_card({
+        '@type' => 'Card',
+        version => '1.0',
+        uid => $id,
+        prodId => $prodid,
+        kind => 'individual',
+        created => $created,
+        name => { full => $name }
+    });
+
+    $self->assert_not_null($card);
```

Comment:
IMO we should never have to assert these in a test.

Reply:
Fixed in new fixup commit to be squashed.